src/sentry/templates/sentry/layout.html

```diff
@@ -41,7 +40,7 @@
   {% block initial_data %}
     {% script %}
     <script>
-      window.__initialData = {% get_react_config %};
+      window.__initialData = {{ react_config|to_json }};
```

Comment:
Will `to_json` also html escape? Putting `</script>` into a field that is serialized here would be a good way to check.

Reply:
It does: https://github.com/getsentry/sentry/blob/81f642933cb92f380698632c379240618f53d281/src/sentry/templatetags/sentry_helpers.py#L28-L30